xbmc/addons/binary-addons/AddonDll.h

```diff
@@ -118,7 +124,7 @@ namespace ADDON
     DllAddon* m_pDll;
     bool m_initialized;
     bool LoadDll();
-    std::map<std::string, std::pair<ADDON_TYPE, KODI_HANDLE>> m_usedInstances;
+    std::unordered_map<PSEUDO_INSTANCE_CLASS, std::pair<ADDON_TYPE, KODI_HANDLE>> m_usedInstances;
```

Comment:
This affects performance when calling find(), for what is the change necessary?

Reply:
I know, but it have the find only during destruction of the instance (where it have more time), during creation of instance and insert for there becomes the sort prevented and added on back.
It does not change much, thought only why not and faster is faster.

But is not a place where we can speak about much entries, currently I think max. 3 entries, maybe in future to 10.

Reply:
ok

Reply:
But: inserting is also slower, because std::map has to find for duplicates first before insertion occurs. I know it does not change much, but I don't see any reason for changing it to unordered.

Reply:
Commit becomes removed :simple_smile: